support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.tsx

```diff
@@ -130,34 +95,37 @@ function mapStateToProps(state: WithDeliveryCheckoutState) {
 		isTestUser: state.page.checkout.isTestUser,
 		country: state.common.internationalisation.countryId,
 		csrf: state.page.csrf,
-		currencyId: currencyFromCountryCode(deliveryAddress.fields.country),
+		currencyId:
+			currencyFromCountryCode(deliveryAddress.fields.country) ??
+			state.common.internationalisation.defaultCurrency,
 		payPalHasLoaded: state.page.checkout.payPalHasLoaded,
 	};
 }
 
+// ----- Map Dispatch/Props ----- //
 function mapDispatchToProps() {
 	const { setCountry } = addressActionCreatorsFor('billing');
 	return {
 		...formActionCreators,
 		fetchAndStoreUserType:
-			(email) =>
+			(email: string) =>
 			(
 				dispatch: Dispatch<Action>,
 				getState: () => WithDeliveryCheckoutState,
 			) => {
 				fetchAndStoreUserType(email)(dispatch, getState);
 			},
 		formIsValid:
-			() =>
-			(dispatch: Dispatch<Action>, getState: () => WithDeliveryCheckoutState) =>
+			() => (_: Dispatch<Action>, getState: () => WithDeliveryCheckoutState) =>
```

Comment:
the first argument (FKA as `dispatch`) is not used, an underscore indicates that in Typescript and means we don't get an unused argument error.